cmd/snapctl/flags.go

```diff
@@ -90,7 +90,7 @@ var (
 
 	flTaskSchedInterval = cli.StringFlag{
 		Name:  "interval, i",
-		Usage: "Interval for the task schedule [ex: 250ms, 1s, 30m]",
+		Usage: "Interval for the task schedule [ex (simple schedule): 250ms, 1s, 30m (cron schedule): \"0 * * * * *\"]",
```

Comment:
Disregard the above comment for now. After thinking about this more, interval may be ok.